.github/workflows/buf-checks.yml

```diff
@@ -1,5 +1,8 @@
 name: Linting and Backwards-Compatibility Checks
-on: [ push ]
+on:
+    push:
+      branches:
+        - master
```

Comment:
What you've got here won't run ever until it hits master, i think you still want it to run on commits to opened PRs, right? I think this is what we want:
```
on: [pull_request]
```

Reply:
Correct, only want it to run on open PRs. Fixed in latest commit.